libs/sql-schema-describer/src/mysql.rs

```diff
@@ -716,7 +716,10 @@ fn extract_precision(input: &str) -> Option<u32> {
 fn extract_enum_values(full_data_type: &&str) -> Vec<String> {
     let len = &full_data_type.len() - 1;
     let vals = &full_data_type[5..len];
-    vals.split(',').map(|v| unquote_string(v)).collect()
+    let mut vals: Vec<_> = vals.split(',').map(|v| unquote_string(v)).collect();
+
+    vals.sort();
```

Comment:
Is there a way to get these values in the order they were defined, instead of lexicographic order?

Reply:
As you can see, they come as a string. And this is different to PostgreSQL where we order them by name.

Reply:
We decided to try to represent the order in which they were listed in the declaration instead of ordering by name.